media-gfx/openscad/openscad-2021.01-r3.ebuild

```diff
@@ -61,6 +61,12 @@ BDEPEND="
 PATCHES=(
 	"${FILESDIR}"/${P}-0001-Gentoo-specific-Disable-ccache-building.patch
 	"${FILESDIR}"/${P}-0002-fix-to-find-lib3mf-2.patch
+	"${FILESDIR}"/${P}-CVE-2022-0496-Out-of-bounds-memory-access-in-DXF-loa.patch
```

Comment:
The CVE fixes need a revbump as they affect runtime behaviour.

Reply:
done